app/controllers/admin/groups_controller.rb

```diff
@@ -0,0 +1,65 @@
+class Admin::GroupsController < Admin::BaseController
+
```

Comment:
If `authorize` is used in all actions it is a good idea to put an after action to make sure that the authorization is always done:

```ruby
after_action :verify_authorized
```